packages/components/README.md

```diff
@@ -27,8 +27,13 @@ export default function MyButton() {
 }
 ```
 
+[Note: is the following paragraph still relevant?]
```

Comment:
As long as some components still use SCSS files, we need it. In the long run, I assume Emotion would handle all styles and it might become obsolete.

Reply:
Done in [`08cd356` (#33960)](https://github.com/WordPress/gutenberg/pull/33960/commits/08cd3568541b4a63fc814d9c99b2c47e1c89ed32)